src/python/pants/util/tarutil.py

```diff
@@ -6,77 +6,77 @@
                         unicode_literals, with_statement)
 
 import tarfile
+from builtins import str
 
-import six
+from future.utils import implements_iterator
 
 
-if six.PY2:
```

Comment:
Now creating the same class for both Py2 and Py3. I'm not sure why the original author only patched the Py2 behavior - Py3 still needs the patch (see new line 64).